app/javascript/components/__test__/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Enzyme, { mount } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import App from '../App'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+it('SearchNew renders without crashing', () => {
+  const div = document.createElement('div')
+  ReactDOM.render(<SearchNew />, div)
```

Comment:
Is there a reason this is testing SearchNew, but it looks like the test for App, and you got rid of the test file for SearchNew?